packages/react/src/components/sideNavigation/mainLevel/MainLevel.module.scss

```diff
@@ -0,0 +1,48 @@
+.mainLevel {
+  composes: level from '../SideNavigation.module.scss';
+
+  font-size: var(--fontsize-body-m);
+
+  & > a,
+  & > button {
+    font-weight: 500;
+  }
+}
+
+.active {
+  composes: activeLevel from '../SideNavigation.module.scss';
+}
+
+.mainLevelListMenu {
+  display: none;
+}
+
+.mainLevelWithDivider {
+  composes: mainLevelWithDivider from '../SideNavigation.module.scss';
+}
+
+.leftIcon,
+.label {
+  margin-right: var(--spacing-xs);
+}
+
+.leftIcon,
+.rightIcon {
+  width: var(--side-navigation-icon-size);
+}
+
+.rightIcon {
+  margin-left: auto;
+}
+
+.mainLevel.open {
+  .arrowIcon {
+    transform: rotate(180deg);
```

Comment:
Alternative to this might be to just apply another arrow icon in the tsx file. There should be arrow icons for all main directions.

Reply:
👍 
- [ ] Replace CSS rotations with up and down icons